src/effects/VST3/VST3Effect.cpp

```diff
@@ -181,18 +181,41 @@ bool VST3Effect::SupportsAutomation() const
    return true;
 }
 
-bool VST3Effect::SaveSettings(const EffectSettings&, CommandParameters& parms) const
+bool VST3Effect::SaveSettings(const EffectSettings& settings, CommandParameters& parms) const
```

Comment:
The commit "Attempt to address..." does not compile, but the following commit repairs that